main/res/values-de/strings.xml

```diff
@@ -19,24 +19,24 @@
   <string name="mystery">Mystery-Cache</string>
   <string name="letterbox">Letterbox Hybrid</string>
   <string name="event">Event-Cache</string>
-  <string name="mega">Mega Event-Cache</string>
-  <string name="giga">Giga-Event Cache</string>
+  <string name="mega">Mega-Event-Cache</string>
+  <string name="giga">Giga-Event-Cache</string>
   <string name="earth">Earth-Cache</string>
-  <string name="cito">Cache in Trash out Event</string>
+  <string name="cito">Cache-In-Trash-Out-Event</string>
   <string name="webcam">Webcam-Cache</string>
   <string name="virtual">Virtual-Cache</string>
   <string name="wherigo">Wherigo-Cache</string>
   <string name="lostfound">Lost &amp; Found Event Cache</string>
-  <string name="ape">Project Ape-Cache</string>
+  <string name="ape">Project-APE-Cache</string>
   <string name="gchq">Groundspeak HQ</string>
   <string name="gps">GPS Adventures Exhibit</string>
   <string name="block">Groundspeak Block Party</string>
   <string name="unknown">Unbekannter Typ</string>
   <!-- cache sizes -->
-  <string name="cache_size_micro">Micro</string>
-  <string name="cache_size_small">Small</string>
-  <string name="cache_size_regular">Regular</string>
-  <string name="cache_size_large">Large</string>
+  <string name="cache_size_micro">Mikro</string>
+  <string name="cache_size_small">Klein</string>
+  <string name="cache_size_regular">Normal</string>
+  <string name="cache_size_large">Groß</string>
```

Comment:
I'm very much in favor of translating the cache sizes. Some reasons

- Geocaching.com translates cache sizes to German
- Opencaching.com translates cache sizes to German
- Garmin devices translates cache sizes to German
- 17 out of 20 other translations translate cache sizes to the native language
- Even the german translation partly translated cache sizes ("other" → "Sonstiges")

In #445, the only argument against translating it is that "the translation would add ambiguity", which I don't understand.

I know many people who use the  German as well as the English terms. However, we are talking about a translation from English to German, so we clearly should translate them (since they are common words and not proper names).